lessons/pythonClasses/lesson.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Python Classes
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# Introduction to Classes in Python
+
+ - **Authors**: Lina Tran
+ - **Research field**: Neuroscience
+ - **Lesson topic**: Object Oriented Programming with Python
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/pythonClasses>
+
+## Topics Covered:
+
+- Defining a Class
+- Creating Instances of a Class
+- Attributes
+  - Class Attributes vs. Normal Attributes
+- Methods
+  - Static Methods
+  - Class Methods
+- Inheritance
+
+### What You Will Need
+
+- Python 2.7, or 3.3+ installed
```

Comment:
Yeah, I should add it's only for those using the jupyter notebook that requires this if choosing python3